Modification concerning the macOS (#23)
NextronSystems/asgard-manual: administration/agent.rst

macOS Agent Deployment
^^^^^^^^^^^^^^^^^^^^^^

To install the agent on macOS, you can just run the PKG file or execute the following command in terminal:

.. code-block:: console
   
     MacBook-Pro:~ nextron$  sudo installer -pkg  /home/nextron/Downloads/asgard2-agent-macos-amd64.pkg -target /

Starting with macOS Big Sur (v11.0), Apple requires software developers
to notarize applications. Our ``asgard2-agent`` installer is notarized.

You can test this by exectuing the follow commands in terminal:

.. code-block:: console
   
   MacBook-Pro:~ nextron$  pkgutil --check-signature /home/nextron/Downloads/asgard2-agent-macos-amd64.pkg
   Package "asgard2-agent-macos-arm64.pkg":
   Status: signed by a developer certificate issued by Apple for distribution
   Notarization: trusted by the Apple notary service
   Signed with a trusted timestamp on: XXXX-XX-XX XX:XX:XX +0000
   ...

> If you are facing issues concerning the installation, please see this section:|br|
> :ref:`appendix/gatekeeper:Bypass Apple verification during installation of asgard2-agent`

Reviewer: ```suggestion
If you are facing issues concerning the installation, please have a look in the chapter
:ref:`appendix/gatekeeper:Bypass Apple verification during installation of asgard2-agent`.
```

   :scale: 40
   :alt: macOS 13 Full Disk Access

> If you need to grant Full Disk Access via MDM, please take a look at this appendix:|br|
> :ref:`appendix/mdm-fulldiskaccess:Full Disk Access for macOS asgard2-agent-service via MDM`

Reviewer: ```suggestion
If you need to grant Full Disk Access via MDM, please have a look at the chapter
:ref:`appendix/mdm-fulldiskaccess:Full Disk Access for macOS asgard2-agent-service via MDM`.
```

.. note:: 
   There is no workaround to this step, since it is an integral
   part of the security design of Apple devices. If you are having trouble
   with THOR scans via ASGARD on macOS, please check if the ``Full Disk
   Access`` permission for the ASGARD agent was granted. Since macOS Mojave
   (v10.14), you need to grant the same permissions to removable volumes,
   if you plan on scanning those.

> 
> .. |br| raw:: html
> 
>       </br>

Reviewer: ```suggestion
```

We dont need the raw html since we try to avoid using it.

Author: This was the only way I found to put the breaks where I want to have them ...
